openstack/identity/v3/policies/requests.go

```diff
@@ -99,3 +99,15 @@ func Create(client *gophercloud.ServiceClient, opts CreateOptsBuilder) (r Create
 	})
 	return
 }
+
+// Get retrieves details on a single policy, by ID.
+func Get(client *gophercloud.ServiceClient, id string) (r GetResult) {
```

Comment:
Small nit: change `id` to `policyID` to match the other occurrences of `policyID`.